src/components/Accordion/Item/Trigger.tsx

```diff
@@ -9,24 +9,31 @@ import { Button } from "../../Button";
 import { ChervonDownIcon } from "../../Icons";
 import { icon } from "../common.css";
 
-export type AccordionProps = PropsWithBox<{
+export type AccordionTriggerProps = PropsWithBox<{
   children: ReactNode;
   buttonDataTestId?: string;
 }>;
 
 export const Trigger = ({
   children,
   buttonDataTestId,
+  disabled,
   ...rest
-}: AccordionProps) => (
+}: AccordionTriggerProps) => (
   <AccordionHeader asChild>
-    <AccordionTrigger asChild>
+    <AccordionTrigger
+      asChild
+      onClick={(e) => {
+        disabled && e.preventDefault();
+      }}
```

Comment:
question: Is not enough to add disabled to AccordionItem ?

Reply:
nope - I tried to do that but I think it did not work as we are using custom `Item` component with `asChild` prop so I had to hack it in this way